repair/row_level.cc

```diff
@@ -767,6 +769,10 @@ class repair_meta {
     bool _small_table_optimization_tm_calculated = false;
     service::frozen_topology_guard _frozen_topology_guard;
     service::topology_guard _topology_guard;
+    // Max fragments size and number are allowed in a single repair_row_on_wire
+    // which is sent as a rpc stream message
+    static constexpr size_t _max_fragments_size = 1024 * 1024;
```

Comment:
Any reason not to reduce it even more? Suggest 100k so there's some spare to reach 128k.

These are still streamed, yes? so there's no round-trip latency * the number of messages.

Reply:
Each such message contains a partition key, I want to batch more fragments for such message. In most of the time, a reasonable total size of fragments allowed in a message would contain all the fragments needed to sync. If the number of fragments is huge, the partition key per such message is not that bad.  

Btw, we use std::list for the frozen_mutation_fragment stored in partition_key_and_mutation_fragments to avoid large allocations. 

```
class partition_key_and_mutation_fragments {
    partition_key _key;
    utils::chunked_vector<frozen_mutation_fragment> _mfs;
}
```

Will the 128K thing have impact at all?

Reply:
We have `partition_key_and_mutation_fragments_v2 { optional<partition_key>, vector<fmf> };` Or just send a stream of mutation fragments.

1MB messages can cause stalls during allocation during compression. Let's reduce it to 240kB.